cli/tests/testdata/test/uncaught_errors.out

```diff
@@ -53,6 +54,6 @@ bar 2 => ./test/uncaught_errors_2.ts:3:6
 bar 3 => ./test/uncaught_errors_2.ts:6:6
 ./test/uncaught_errors_3.ts (uncaught error)
 
-FAILED | 2 passed | 5 failed ([WILDCARD])
```

Comment:
@nayeemrmn seems like this is flaky now: https://github.com/denoland/deno/runs/7362087847?check_suite_focus=true